DEPS

```diff
@@ -63,6 +63,12 @@ hooks = [
     'condition': 'checkout_ios and host_os == "mac"',
     'action': ['python3', 'script/ios_bootstrap.py']
   },
+  {
+    # The .env file is used by `gn gen`. Create it if it doesn't exist.
+    'name': 'ensure_env_config',
+    'pattern': '.',
+    'action': ['python3', 'build/env_config.py', 'create_if_not_found', '.env'],
```

Comment:
Could we just call:
`python3 -c 'from pathlib import Path;Path(".env").touch()'`